plugins/vpp/ifplugin/ifaceidx/dhcp_index_test.go

```diff
@@ -38,7 +38,7 @@ func dhcpTestInitialization(t *testing.T) (idxvpp.NameToIdxRW, ifaceidx.DhcpInde
 	}
 
 	// initialize index
-	nameToIdx := nametoidx.NewNameToIdx(logrus.DefaultLogger(), "testName", "dhcp_index_test", ifaceidx.IndexDHCPMetadata)
+	nameToIdx := nametoidx.NewNameToIdx(logrus.DefaultLogger(), "testName", ifaceidx.IndexDHCPMetadata)
```

Comment:
you should keep dhcp_index_test and remove testName, but it doesn't really matter